src/views/addMovieView.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { addMovieToStore } from '../actions/storedMovies';
+import { processInputChange, resetForm } from '../actions/form';
+
+import countries from '../common/countries';
+import { API_PARAMS } from '../common/constants'
+
+const {MOVIE_DETAILS} = API_PARAMS;
+
+class AddMovieView extends Component {
+
+    handleChange = (e) => {
+        let value = e.target.value;
+        const key = e.target.name;
+
+        if (e.target.hasAttribute('multiple')) {
+            value = Array.from(e.target.selectedOptions).map(item => item.value).join(", ");
```

Comment:
As you save in line 16 the reference to ```e.target.value``` in ```value``` var, you just mutate the ```e.target.value``` here. Thats something we need to avoid.